agent/unit-test/softwarecontaineragent_unittest.cpp

```diff
@@ -37,28 +37,49 @@ class SoftwareContainerAgentTest: public ::testing::Test
     std::shared_ptr<Workspace> workspace;
 
     void SetUp() override
-    {
+            {
         sca = new SoftwareContainerAgent(
-                      m_context
-                    , m_preloadCount
-                    , m_shutdownContainers
-                    , m_shutdownTimeout);
+                m_context
```

Comment:
I'd move the , to the end of the lines as well, we changed that style a few weeks ago as it makes more sense.